src/interfaces/assistants_web/src/app/layout.tsx

```diff
@@ -13,7 +13,7 @@ export const metadata: Metadata = {
 const Layout: React.FC<React.PropsWithChildren> = ({ children }) => {
   return (
     <html lang="en" className="dark">
-      <body>
+      <body className="text-volcanic-100 dark:text-marble-950">
```

Comment:
we will need to update so many components now that don't require this classes anymore

Reply:
done